bundle/src/main/java/com/adobe/acs/commons/ccvar/util/ContentVariableReplacementUtil.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2021 Adobe
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.adobe.acs.commons.ccvar.util;
+
+import com.adobe.acs.commons.ccvar.TransformAction;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static com.adobe.acs.commons.ccvar.impl.PropertyConfigServiceImpl.PARSER_SEPARATOR;
+
+/**
+ * Util class used to provide helper methods for finding and replacing the tokens used in this feature.
+ */
+public class ContentVariableReplacementUtil {
+
+    private static final Pattern PLACEHOLDER_PATTERN = Pattern.compile("\\{\\{([a-zA-Z0-9_:\\-]+\\.[a-zA-Z0-9_:\\-]+(\\|[a-zA-Z0-9_:\\-]*)?)}}");
+    private static final Map<String, String> REQUIRED_ESCAPE = escapeMap();
+    private static final String PLACEHOLDER_BEGIN = "{{";
```

Comment:
The characters {{ and }} are not allowed in HTL values that represent uri's, so unfortunately tokens get stripped out before they can be translated.

We will need to use something other than {{ and }}, such as (( and ))

`PLACEHOLDER_PATTERN` needs a corresponding update.